java/google/registry/keyring/api/DummyKeyringModule.java

```diff
@@ -42,6 +42,30 @@
  * with all attempted outgoing connections failing because the supplied dummy credentials aren't
  * valid. For a real system that needs to connect with external services, you should replace this
  * module with one that loads real credentials from secure sources.
+ * 
+ * <p>The dummy PGP keyrings are created using gnupg1/pgp1 roughly like the following (using
+ * gnupg2/pgp2 is an exercise left for the developer):
+ * <code>
+ * {@code
```

Comment:
@code and \<code> do the same thing, so don't use both.  I think what you want here is:

\<pre>{@ code
   ....
}\</pre>

Reply:
Yes. Thanks for catching that.